src/checks.ts

```diff
@@ -109,8 +109,16 @@ export async function checkTsconfig(dirPath: string, dt: DefinitelyTypedInfo | u
         throw new Error('Must specify "lib", usually to `"lib": ["es6"]` or `"lib": ["es6", "dom"]`.');
     }
 
+    if (!options.strict) {
```

Comment:
> Note that there’s no package in DefinitelyTyped with strict set to false.

In that case, wouldn't it be better to have

```ts
if ("strict" in options) {
  if (!options.strict) {
    throw new Error("...")
  }
}
else {
  // (new noImplicitAny code here)
  // (existing noImplicitThis et al code here)
```

Reply:
I’m doing this in <https://github.com/microsoft/dtslint/pull/277>, because this PR depends on all **DefinitelyTyped** packages having `"noImplicitAny": true`, which will require fixing the remaining 49 packages that won’t yet be fixed by <https://github.com/DefinitelyTyped/DefinitelyTyped/issues/42682>.